### DISCUSSION TITLE
LavaSearch with wavelink

### DISCUSSION THREAD

**Discussion by Jonas0o0** [EvieePy/Wavelink#315] [Q&A]
I install lavaSearch and i try this for find playmist:

```Python
tracks: wavelink.Search = await wavelink.Playable.search(query, types=f"playlist", source="spotify")
```

but I have a error 

```Python
exception: TypeError: Playable.search() got an unexpected keyword argument 'types'
```

So how we use LavaSearh plugin with wavelink ?

**Comment by chillymosh** [comment]
We do not have a implementation to interface with LavaSearch. Your linter should also be highlighting that `type` is not a kwarg for `Playable.search()`, hence why you get an exception.

We do have LavaSrc integration.

LavaSearch is something we have discussed adding support for, but it will require some further rework to allow types and the other endpoints in a meaningful manner.

**Reply by Jonas0o0** [reply]
So I can't use LavaSearch ?

**Comment by Jonas0o0** [comment]
And how do I search for a track in Spotify, I try that but it doesn't work: 

```Python
tracks: wavelink.Search = await wavelink.Playable.search(query, source="Spotify")
```
